.travis.yml

```diff
@@ -26,7 +28,7 @@ install:
   - source activate test-environment
   - conda install -c conda-forge aipy pycodestyle
   - pip install coveralls
-  - pip install git+https://github.com/HERA-Team/pyuvdata.git@v1.3
+  - pip install git+https://github.com/HERA-Team/pyuvdata.git@9c94be0366eb41692f5501caac99981026a7ffe6
```

Comment:
Can't we get up to date with master? I'd like to try to always be up to date with master and only build against a specific version or commit when master breaks us.